src/app/ApplicationTemplate.Client/Framework/Serialization/SystemTextJsonToContentSerializerAdapter.cs

```diff
@@ -37,12 +39,17 @@ public async Task<T> DeserializeAsync<T>(HttpContent content)
 		}
 	}
 
-	public Task<HttpContent> SerializeAsync<T>(T item)
+	public string GetFieldNameForProperty(PropertyInfo propertyInfo)
+	{
+		throw new NotImplementedException();
```

Comment:
In a project where we're actually using this custom implementation, we went with the refit implementation https://github.com/reactiveui/refit/blob/35965b8238eaf237af5305312989a2f66246491d/Refit/SystemTextJsonContentSerializer.cs#L60